docs/zk_rollups_after_eip4844.md

```diff
@@ -152,41 +153,35 @@ err = ethclient.Client.SendTransaction(context, signedTx)
 The pseudocode above is for illustration purposes only. [Here](https://www.rareskills.io/post/ethclient-golang) is a more complete
 implementation.
 
-### Update the verification contract
-
-For brevity, assume that the `verify` function is called as a blob transaction, so we can use the `blobhash` opcode to get the
-versioned hash of a blob.
+### How To Verify The Commitment
 
-Prover now needs to send an additional proof `blobKZGProof` and the `proof` must contain the blob consistency proof. More details in
-the pseudo code below:
+Assume that in the `verify` function, we want to check whether the parameter `blobKZGProof` is consistent with the underlying blob. We
+can first use the `blobhash` opcode to get the versioned hash of the blob, then use `point_evaluation_precompile()` to verify the
+consistency as shown below.
 
 ```Solidity
 /// Memory layout of `blobKZGProof`:
 /// | z       | y       | kzg_commitment | kzg_proof |
 /// |---------|---------|----------------|-----------|
 /// | bytes32 | bytes32 | bytes48        | bytes48   |
-///
-/// `proof` must contain the blob consistency proof.
-function verify(
-  bytes calldata blobKZGProof,
-  bytes calldata proof
-) external {
-    // Calculate the versioned hash of blobs[0]
-    bytes32 _blobVersionedHash = blobhash(0);
-
-    // Call the point evaluation precompile to ensure that `blobKZGProof` corresponds to the blob data.
-    {
-        (bool success, bytes memory data) = POINT_EVALUATION_PRECOMPILE_ADDR.staticcall(
-            abi.encodePacked(_blobVersionedHash, blobKZGProof)
-        );
-        // We verify that the point evaluation precompile call was successful by testing the latter 32 bytes of the
-        // response is equal to BLS_MODULUS as defined in https://eips.ethereum.org/EIPS/eip-4844#point-evaluation-precompile
-        if (!success) revert ErrorCallPointEvaluationPrecompileFailed();
-        (, uint256 result) = abi.decode(data, (uint256, uint256));
-        if (result != BLS_MODULUS) revert ErrorUnexpectedPointEvaluationPrecompileOutput();
+    function verify(
```

Comment:
align code indent

Reply:
leave this as is as there is no global function level in solidity (meaning it has to be inside a class, thus leading to the indentation)

Reply:
However, the comment and the function should have the same indentation.

Reply:
You can change it if you want but for me it looks fine